tests/integration/pilot/tunneling/forward-proxy/deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
```

Comment:
fwiw for unrelated reasons I have been working on adding CONNECT support in the echo app. So we could probably replace this long term. No problem for short term

Reply:
Ok, I will get back to it once the CONNECT in echo is ready.